### ISSUE TITLE
border-image: none; produce an error

### ISSUE THREAD

**Issue by bouzlibop** [bjankord/stylelint-config-sass-guidelines#16]
As for now there is [this rule](https://github.com/bjankord/stylelint-config-sass-guidelines/blob/master/src/.stylelintrc.json#L23-L27) defined:
```
    "declaration-property-value-blacklist": {
      "/^border/": [
        "none"
      ]
    },
```

If I want to define something like this:
`border-image: none;`
it will produce an error because of the above mentioned rule.

I'm curious why `stylelint-config-sass-guidelines` disallow it?

**Comment by abdennour** [comment]
The same . Why this rule , namely border-style accepts none according to [W3C](https://www.w3schools.com/css/css_border.asp) .

As a hot fix, I disabled this rule by copying the same rule but without value , which means without blacklist .
```
{
  "extends": ["stylelint-config-sass-guidelines"],
  "rules" : {
    "declaration-property-value-blacklist": {
      "/^border/": []
    }
  }
}


```

**Comment by bjankord** [comment]
This config is based on https://sass-guidelin.es [lint rules](https://sass-guidelin.es/#scss-lint), which uses [SCSS Lint's BorderZero rule](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#borderzero).

If you find that you do not want to enable the rule, extending the stylelint-config and overriding the rule like @abdennour did is the way I'd recommend.

**Comment by bouzlibop** [comment]
@bjankord thanks for the answer. I still think we need a fix to this rule. Please take a look:

You mentioned that config is based on sass guidelines, which uses SCASS Lint's BorderZero rule. The thing is that this rule as it is defined here: https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/border_zero.rb  forbids `none` value for:
```
border
border-top
border-right
border-bottom
border-left
```

As you can see it **is allowed** to use `none` for `border-image` for example. It is not defined in the blacklist. You can't use `none` for `border` itself, but you can use it for `border-image`.

My point is that [current regex](https://github.com/bjankord/stylelint-config-sass-guidelines/blob/master/src/.stylelintrc.json#L21) (`/^border/`) is probably wrong if we want to follow sass guidelines exactly. We should disallow only:
```
border
border-top
border-right
border-bottom
border-left
```
and not everything that begins with `border`.

**Comment by bjankord** [comment]
That's a good point, I'm open to a PR to correct this.

**Comment by bjankord** [comment]
@bouzlibop This has been released in [v4.1.0](https://github.com/bjankord/stylelint-config-sass-guidelines/releases/tag/v4.1.0)

Thank you for the contribution!

**Comment by bouzlibop** [comment]
Thank you! :)
